docs/release.md

```diff
@@ -60,6 +60,8 @@ The following sections are generally for the maintainers of XDP-for-Windows. The
 
 > **Note** - TODO
 
+* Authorize the new branch in the OneBranch.Official internal pipeline ([Manage Build Pipeline Authorized Branches](https://mscodehub.visualstudio.com/WindowsXDP/_apps/hub/EZStart.management-ux.onebranch-resources#authorizedbranches/xdp/2407)) under OneBranch Resources.
```

Comment:
Maybe add a note that both Official and PullRequest pipelines should be authorized.

Reply:
Oh, I didn't realize we needed to include PullRequest too. Good thing we're writing this down.